stable/main.pony

```diff
@@ -63,9 +63,22 @@ actor Main
       ""
     end
     try
-      Shell.from_array(
-        ["env"; "PONYPATH="+ponypath].>append(rest), env~exitcode()
-      )?
+      ifdef windows then
+        var cmd: String trn = recover String end
+        cmd.append("cmd /C \"set \"PONYPATH=")
+        cmd.append(ponypath)
+        cmd.append("\" &&")
+        for arg in rest.values() do
+          cmd.append(" ")
+          cmd.append(arg)
```

Comment:
Does windows need any escaping here if the argument contains special characters, like a space?